lib/versioning/node/index.ts

```diff
@@ -1,5 +1,8 @@
+import moment from 'moment';
+import nodeData from '../../../data/node-js-schedule.json';
```

Comment:
please no static import, use
```ts
  const file = await resolveFile('data/node-js-schedule.json');
  const nodeJsSchedule = (await import(file)) as NodeJsData;
```

Maybe we need an optional async init function for static versioning initialization?

Reply:
Later we can parse Ubuntu release info as well... And we cannot use top-level awaits, as I can see.

Reply:
So, are we about to introduce optional `async init()` method on `VersioningApi` interface?

Reply:
yes, because static import would blocking renovatebot/docker-renovate#26, so we would never be able to bundle.
thats why i suggest a new optional async init function for versionings, where we can do this initialization.
Maybe platforms manager and datasource can benefit from those api too?

@rarkins what do you think?

Reply:
I don't mind, however I would prefer if we could import this JSON file synchronously so we don't need a special case for it in `data/`. Even considered massaging it into a JS/TS file if we have to.

Reply:
if we enable typescript json interop, tsc will copy the file to the dist folder. but then we need to read our package,json for version async, otherwise tsc will copy that file too, without required version bump

Reply:
Okay, I guess I'll make separate PR for this, but you can take a look at latest commit for general idea

Reply:
As an temporary alternative, we can hardcode this Node releases here and open an issue for module initializers